src/transformState.js

```diff
@@ -6,6 +6,27 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    if (action.type === 'clear') {
+      Object.keys(state).forEach(key => {
+        delete state[key];
+      });
+    }
+
+    if (action.type === 'removeProperties') {
+      action.keysToRemove.forEach(key => {
+        delete state[key];
+      });
+    }
+
+    if (action.type === 'addProperties') {
+      for (const [key, value] of Object.entries(action.extraData)) {
+        state[key] = value;
+      }
+    }
```

Comment:
Check list point 2 > [CODE STYLE]: Use switch statement if you have limited amount of conditions.

Reply:
oh, didn't noticed there is checklist - thx

Reply:
Just like me at the beginning ^^*